doc/manuals/orion-api.md

```diff
@@ -1800,77 +1800,51 @@ Content-Type is `application/json`
-+ `lastFailure` (not editable, only present in GET operations): Last failure timestamp in ISO8601 format.
-  Not present if subscription has never had a problem with notifications.
-+ `lastSuccess` (not editable, only present in GET operations): Timestamp in ISO8601 format for last successful notification.
-  Not present if subscription has never had a successful notification.
+| Parameter                | Optional | Type   | Description                                                                                   |
+|--------------------------|----------|--------|-----------------------------------------------------------------------------------------------|
+| `attrs` or `exceptAttrs` |          | string | Both cannot be used at the same time. <ul><li><code>attrs</code>: List of attributes to be included in notification messages. It also defines the order in which attributes must appear in notifications when <code>attrsFormat</code> <code>value</code> is used (see "Notification Messages" section). An empty list means that all attributes are to be included in notifications. See "Filtering out attributes and metadata" section for more detail.</li><li><code>exceptAttrs</code>: List of attributes to be excluded from the notification message, i.e. a notification message includes all entity attributes except the ones listed in this field.</li><li>If neither <code>attrs</code> nor <code>exceptAttrs</code> is specified, all attributes are included in notifications.</li></ul>|
+| `http` or `httpCustom`   | ✓        | object | One of them must be present, but not both at the same time. It is used to convey parameters for notifications delivered through the HTTP protocol. |
+| `attrsFormat`            | ✓        | string | Specifies how the entities are represented in notifications. Accepted values are `normalized` (default), `keyValues` or `values`.<br> If `attrsFormat` takes any value different than those, an error is raised. See detail in "Notification Messages" section. |
+| `metadata`               | ✓        | string | List of metadata to be included in notification messages. See "Filtering out attributes and metadata" section for more detail. |
+| `timesSent`              |          | number | Not editable, only present in GET operations. Number of notifications sent due to this subscription. |
+| `lastNotification`       |          | string | Not editable, only present in GET operations. Last notification timestamp in ISO8601 format. |
+| `lastFailure`            |          | string | Not editable, only present in GET operations. Last failure timestamp in ISO8601 format. Not present if subscription has never had a problem with notifications. |
+| `lastSuccess`            |          | string | Not editable, only present in GET operations. Timestamp in ISO8601 format for last successful notification.  Not present if subscription has never had a successful notification. |
```

Comment:
We should some way explain that some parameters cannot be used when creating or updating a subscription as they are only used in GET responses to some some status/statistical information.

One solution could be to use the "Optional?" column this way

```suggestion
| `timesSent`              | Only in GET         | number | Not editable, only present in GET operations. Number of notifications sent due to this subscription. |
| `lastNotification`       | Only in GET          | string | Not editable, only present in GET operations. Last notification timestamp in ISO8601 format. |
| `lastFailure`            |  Only in GET        | string | Not editable, only present in GET operations. Last failure timestamp in ISO8601 format. Not present if subscription has never had a problem with notifications. |
| `lastSuccess`            | Only in GET         | string | Not editable, only present in GET operations. Timestamp in ISO8601 format for last successful notification.  Not present if subscription has never had a successful notification. |
```

Although maybe another alternatives are also possible.

Reply:
With regards to optional fields used in creation and updating requests, we have this cases:

1. Fields that are not provided and in the GET responses are not shown
2. Fields that are not provided and in the GET responses use a default value

(I think we started implementing according to style 1, but newer additions, e.g. `onlyChangedAttrs`, use style 2)

Should be distinguish both cases in the "Optional" column some way? Unsure...

Reply:
"Only on retrieval"

Reply:
"GET responses" -> "Only on retrieval"

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4158#:~:text=to%20optional%20col-,f24e36b,-mapedraza%20force%2Dpushed